typescript/src/events/guilds/members/notMutedMemberAddNotify.ts

```diff
@@ -2,18 +2,18 @@ import { GuildSettings } from '#lib/database';
 import { LanguageKeys } from '#lib/i18n/languageKeys';
 import { Colors } from '#lib/types/Constants';
 import { Events } from '#lib/types/Enums';
-import { MessageLogsEnum } from '#utils/constants';
 import { ApplyOptions } from '@sapphire/decorators';
 import { Event, EventOptions } from '@sapphire/framework';
 import { GuildMember, MessageEmbed } from 'discord.js';
 
 @ApplyOptions<EventOptions>({ event: Events.NotMutedMemberAdd })
 export class UserEvent extends Event {
 	public async run(member: GuildMember) {
-		const [enabled, t] = await member.guild.readSettings((settings) => [settings[GuildSettings.Events.MemberAdd], settings.getLanguage()]);
-		if (!enabled) return;
+		const key = GuildSettings.Channels.Logs.MemberAdd;
+		const [channelId, t] = await member.guild.readSettings((settings) => [settings[key], settings.getLanguage()]);
```

Comment:
```suggestion
		const [logChannelId, t] = await member.guild.readSettings((settings) => [settings[key], settings.getLanguage()]);
```

And rename references accordingly. I had to check what the heck `channelId` was, a simple other name makes it easier already.